tools/recetox_aplcms/recetox_aplcms_hybrid.xml

```diff
@@ -67,9 +67,9 @@
 
     <tests>
         <test>
-            <param name="files" value="mbr_test0.mzml,mbr_test1.mzml,mbr_test2.mzml" ftype="mzml"/>
-            <param name="known_table" value="known_table.parquet" ftype="parquet"/>
-            <output name="recovered_feature_sample_table" file="recovered_feature_sample_table.parquet" ftype="parquet" compare="sim_size" delta="1000"/>
+            <param name="known_table" value="known_table.parquet" ftype="parquet" />
+            <param name="files" value="mbr_test0.mzml,mbr_test1.mzml,mbr_test2.mzml" ftype="mzml" />
+            <output name="recovered_feature_sample_table" file="hybrid_recovered_feature_sample_table.parquet" ftype="parquet" compare="sim_size" delta="1000" />

```

Comment:
Is there any other way than comparing with similar size, since it actually doesn't check the content of the file?

Reply:
I'd like to keep this PR to a minimum, so I can run performance tests over the weekend. So yes, but let's implement it in a subsequent PR.

Reply:
Ok, it would be simply about the content. I'll push it in a new issue.